src/components/views/rooms/wysiwyg_composer/WysiwygComposer.tsx

```diff
@@ -51,6 +55,32 @@ export function WysiwygComposer(
         ref.current?.focus();
     }, [content, mxClient, roomContext, wysiwyg, props, ref]);
 
+    useDispatcher(defaultDispatcher, (payload: ActionPayload) => {
```

Comment:
This hook is a bit massive, if you can make a custom hook from it, it'll be great!